mkdocs.yml

```diff
@@ -36,6 +36,9 @@ plugins:
 
 nav:
     - Home: index.md
-    - Tasks: tasks.md
-    - Flows: flows.md
+    - Cloud:
+      - Credentials: cloud/credentials.md
```

Comment:
Does dbt core and cloud use separate credentials?

Reply:
The main different in credentials is that dbt Cloud will use API service tokens while dbt Core will use `profiles.yml` credentials.